stack-graphs/src/graph.rs

```diff
@@ -1323,6 +1324,14 @@ impl StackGraph {
             None => Either::Left(std::iter::empty()),
         }
     }
+
+    /// Returns an iterator of all of the edges that begin at a particular source node.
+    pub fn incoming_edge_count(&self, sink: Handle<Node>) -> usize {
+        match self.incoming_edges.get(sink) {
+            Some(count) => *count,
+            None => 0,
+        }
```

Comment:
You can simplify using [`Option::unwrap_or_default`](https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap_or_default)

Reply:
That's a nice one. Also `unwrap_or` would've been cleaner. But this will go, because of not including the incoming edge count optimization.